modules/pubmaticClientBidAdapter.js

```diff
@@ -0,0 +1,328 @@
+import * as utils from 'src/utils';
```

Comment:
so this is a copy of the original `pubmaticBidAdapter` ?

Reply:
Kind of, we changed only the BIDDER_CODE to pubmatic_client. This is how we are doing s2s and client side in rubicon as well so we replicated it

Reply:
I see!
We need to put a reminder here somewhere, because when I updated the prebid version, I didn't think of changing the rabicon file, I didn't realise it existed... So all good for this PR, and in my update PR I'll add something about it!